src/main.cpp

```diff
@@ -77,4 +77,4 @@ void loop() // cppcheck-suppress unusedFunction
 
 /******************************************************************************
  * Local functions
- *****************************************************************************/
+ *****************************************************************************/
```

Comment:
Necessary?

Reply:
Not sure what the difference is. Missing line at the end of the file?

Reply:
Revert the change with the help of git. E.g. in TortoiseGit open the log and click on the file and revert just this change.